python/lsst/ts/scheduler/driver.py

```diff
@@ -733,6 +734,17 @@ def register_observation(self, observation):
 
         return target_list
 
+    def cold_start(self, obslist=None):
+        """Rebuilds the state of the scheduler from a list of observations"""
+        self.log.info("Running coldstart (PBS)")
+        f = open("first100pbs","rb")
```

Comment:
This is reading a file with hard-coded name. Shouldn't it use the list of observations that it receives?